worker/src/test/java/gov/cms/ab2d/worker/processor/JobProcessorIntegrationTest.java

```diff
@@ -73,14 +73,15 @@ void processJob() {
         jobRepository.save(job);
         createContract(sponsor);
 
-        var processedJob = cut.process("S001");
+        var processedJob = cut.process("S00000");
 
         assertThat(processedJob.getStatus(), is(JobStatus.SUCCESSFUL));
         assertThat(processedJob.getStatusMessage(), is("100%"));
         assertThat(processedJob.getExpiresAt(), notNullValue());
         assertThat(processedJob.getCompletedAt(), notNullValue());
     }
 
+    @Disabled("disable till the BFD calls are mocked out.")
     @Test
```

Comment:
This test is disabled till the BDF calls are mocked.

Reply:
How long is this one taking?

Reply:
Also, why is it taking long if we have S0000 set to 10 patients only?

Reply:
I do not have the time, but it ran long enough and filled the log and failed the build in CI.
https://travis-ci.org/CMSgov/ab2d/builds/637433959?utm_source=github_status&utm_medium=notification

Reply:
You are right, this should only return 10 patients, but I just realized that I had a randon number appended to the contractNumber in this test case - due to which it was getting a bunch of patient records. Will have a commit shortly to fix it.

Reply:
Once I removed the random number suffix to the contractNumber in this test, it started finishing fast as it was 10 patient records only. So I went back and enabled this test.

Reply:
The CI log is flooded with "Attempting to append to file" and "finished writing" messages. Intensive logging was _likely_ the reason for slowness. If we drop these two log entries as too verbose, then 100 benes would not be a problem as well.

Reply:
I could change the logging level to DEBUG... especially the first one `Attempting to append to file` as it is happening for each `appendToFile`.
The 2nd line you mentioned (`finished writing`) only happens once per patient record. But I could lower that to DEBUG as well.

Reply:
`DEBUG` is fine

Reply:
done - reduced logging level to DEBUG and now it returns 100 patient records for contract `0000` for this test.